js/browser.js

```diff
@@ -66,6 +70,20 @@ var Billing = Class(Emitter, function (supr) {
     }
   };
 
+  this.isPaymentsReady = function () {
+    if (GC.app.payments_ready) {
+      return true;
+    }
+    return false;
+  }
```

Comment:
best way would be to move this function to FBInstant plugin itself. That way we don't have to store state in GC.app.payments_ready and also we don't have to pass the state around